src/pages/AdminManageDarCollections.jsx

```diff
@@ -21,6 +21,51 @@ export default function AdminManageDarCollections() {
   const searchRef = useRef('')
   const filterFn = getSearchFilterFunctions().darCollections
 
+  // Default columns for admin console
+  const baseColumns = [
```

Comment:
Should this new functionality be refactored into a common component that can be used for the multiple pages in this PR?

Reply:
Done!  See https://github.com/DataBiosphere/duos-ui/pull/3033/commits/1f2ebfb01316fc6fdf630f73674db8e29f6c7a58 and https://github.com/DataBiosphere/duos-ui/pull/3033/commits/317cf9e87393dabba9e1dbaa052dbf1f7caef362.